src/resource_server/views/api_v1/api.py

```diff
@@ -16,13 +16,24 @@ def cmd(endpoint):
     base_url = current_app.config["SSH_KEYSIGN_BASE_URL"]
     user = current_user.claims[current_app.config["SSH_PRINCIPAL_CLAIM"]]
 
+    if request.method == 'POST':
+        params = request.json
+    elif request.method == 'GET':
+        params = request.args 
+    else: 
+        params = dict()
+
     # If the schema validator does raise any error then load the right endpoint values
-    command = load_template_values(validate_schema(path_file), endpoint)
+    command = load_template_values(validate_schema(path_file), endpoint, params)
 
     # Check the app env and get the mocking server port
     if current_app.config["TESTING"]:
```

Comment:
can we configure the test config.json to use the correct port instead of testing a parameter that is forbidden in production?

Reply:
The problem that I noticed is that the mock server generates a different port every time. but now with the request params that is not a problem because I can pass it through.

Reply:
ok ... then leave it for now ....
Ideally the config file will be loaded and validated on startup anyway. There is no point in having a running server with an invalid config file.
If the config is loaded on app initialisation then it makes it also possible to update the config in tests, which would allow us to get rid of this, because in your tests you are essentially allowing an invalid parameter this way, which is not ideal.